Sources/ScipioKit/Producer/FrameworkProducer.swift

```diff
@@ -136,6 +144,16 @@ struct FrameworkProducer {
         }
     }
 
+    private func makeBuildOptions(for product: BuildProduct, basedOn baseBuildOptions: BuildOptions, platformMatrix: PlatformMatrix) -> BuildOptions {
+        guard let overriddenSDKs = platformMatrix[product.target.name] else {
```

Comment:
No.  😅 

I thought again, it's better to pass a struct represents packages and targets rather than just target names.

Reply:
I considered it again.

It's not unique. However, same-named targets are contained in multiple dependencies, they would conflicts in the resolving phase. So it may not co-existence together.

As a result, it is unique in actual.